app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -336,4 +341,12 @@ class WebViewActivity : AppCompatActivity(), io.homeassistant.companion.android.
             super.onRequestPermissionsResult(requestCode, permissions, grantResults)
         }
     }
+
+    @TargetApi(28)
```

Comment:
This isn't going to keep lower devices from calling this.  It only fixes the linting error.  You should remove this and add the logic that you add above in here.

https://developer.android.com/reference/android/support/annotation/RequiresApi.html

```
private fun isCutout(): Boolean {
        var cutout = false
        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P && window.decorView.rootWindowInsets.displayCutout != null)
            cutout = true
        return cutout
    }
```

Reply:
en effet, je l'ai ajouté dans isCutout()

Reply:
I'm not sure if you need the `@TargetApi` anymore.  Does it still complain with it removed?  If it does you should still use the constant instead of the number `@TargetApi(Build.VERSION_CODES.P)`

Reply:
No request, i delete it